PennMobile/src/main/java/com/pennapps/labs/pennmobile/viewholders/HomeBaseHolder.kt

```diff
@@ -3,7 +3,7 @@ package com.pennapps.labs.pennmobile.viewholders
 import androidx.recyclerview.widget.RecyclerView
 import com.pennapps.labs.pennmobile.databinding.HomeBaseCardBinding
 
-class HomeBaseHolder(
+open class HomeBaseHolder(
```

Comment:
Can you also make the rest of the view holders inherit from this class for consistency?